python/jupytergis_lab/jupytergis_lab/notebook/gis_document.py

```diff
@@ -46,8 +46,6 @@ class GISDocument(CommWidget):
     :param path: the path to the file that you would like to open. If not provided, a new empty document will be created.
     """
 
-    path: Optional[Path]
-
```

Comment:
We don't know the path when we create an untitled document; it could be `untitled.jGIS` or `untitled99.jGIS`. The Javascript side knows, but how do we get that info into Python land? See #629

Reply:
Maybe we should require the `path` argument in the `GISDocument`, to avoid
- adding a way to display the path of the document in UI
- adding a way to change it (`save as`), which would lead to the same issue as in #595

Reply:
Otherwise the `untitled.jGIS` should probably be created from the kernel itself.

Reply:
Agreed, that's also what I commented in https://github.com/geojupyter/jupytergis/issues/629